CreeDictionary/API/models.py

```diff
@@ -169,6 +181,30 @@ class Wordform(models.Model):
     # pure MD content won't be included
     PREVERB_ASCII_LOOKUP: Dict[str, Set["Wordform"]] = defaultdict(set)
 
+    def get_absolute_url(self) -> str:
```

Comment:
[Django convention](https://docs.djangoproject.com/en/3.0/ref/models/instances/#get-absolute-url) to have this function. But hey! their example in the docs passes smart object in the template!!!

![Screenshot from 2020-02-21 19-50-58](https://user-images.githubusercontent.com/44753941/75085151-84c47b80-54e3-11ea-8ba8-6836678b667d.png)

Reply:
I think I should have been more clear what I meant by smart and dumb objects. The models can be plenty smart and _should_ have methods like `get_absolute_url()`. That said, **immediately before** the model gets passed to a template, it should be _serialized_.

So, there should be some code that converts the model into a dumb object that _includes_ the absolute URL.

Here's some pseudo-code of what I mean:

```python
def my_view(request):
    search_result = Wordform.search(...)
    assert not hasattr(search_result, 'lemma_url'), "this should not be explicitly stored!"
    assert search_result.get_absolute_url(), "this should work!"
    # serialize the result:
    dumb_object = search_result.to_dict()
    assert "lemma_url" in dumb_object
    assert dumb_object["lemma_url"] == search_result.lemma.get_absolute_url()
    return render("my-template.html", search_result=dumb_object)
```

Reply:
our current SearchResult is a NamedTuple, isn't that already serialized?